definitions.d.ts

```diff
@@ -0,0 +1,392 @@
+// Definitions by: Robbie Van Gorkom <https://github.com/vangorra>
+//                 Matthew Zartman <https://github.com/mattmm3d>
+//                 Jan Dvorak <https://github.com/storytellercz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
+
+/// <reference types="meteor" />
```

Comment:
You might need to remove this line for the types to work in apps that are using the types from core packages instead of @types/meteor.

Reply:
Removed.